backend/src/index.js

```diff
@@ -8,13 +8,16 @@ const server = require('http').Server(app);
 
 const config = require('./config');
 
+// setup database connexion
+require('./config/mongoose');
+
 app.use('/api', require('./api'));
 
 app.use(serveStatic('./public'));
 
-app.get('/*', (req, res) => {
+/* app.get('/*', (req, res) => {
```

Comment:
Just curious, why comment this? You will need it if you ever use frontend routing.